docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -391,6 +391,12 @@ The unique identifier for the cluster.
 
 Default: `""`
 
+##### --force_global_transactions
+
+If true, forces all transactions through this instance to always be global transactions that use the `system.transactions` transaction status table.
```

Comment:
Do you want to add the note about latency here, as well? I think it would be a good idea. (And is there anywhere else we mention this setting, or should mention it?)

Reply:
I added a link to the explore page - the flag isn't mentioned elsewhere but just causes the database to act as if the similarly named session variable is true at all times. It could maybe be added as a small note on the explore page, but it might just serve as a distraction from the example -- the ideal case is to never have to set this flag.

Reply:
Link to explore is good. Thanks!

Here in the reference section, I think we **should** call out the latency consideration. Especially since we want to reinforce that this is something you should rarely if ever have to do.